client/lib/feature-detection/index.js

```diff
@@ -0,0 +1,19 @@
+/** @format */
+
+const featureDetection = {
+	/**
+	 * Detects if CSS custom properties are supported
+	 *
+	 * @return {boolean} true when feature is supported
+	 */
+	supportsCssCustomProperties: () => {
+		return (
+			window &&
```

Comment:
I believe (but could be wrong) that this would throw a ref error if used in SSR or some other non-browser situation.
Instead, could we opt for the safer (but uglier 😞) `typeof window !== 'undefined'` as per our guidelines?

```js
function someFunction () {
	return window && notWindow && 'cool!';
}
someFunction ()
// > Uncaught ReferenceError: notWindow is not defined
```

Reply:
I've seen `global.window` checks elsewhere